inc/wc-calypso-bridge-products.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Adds additional meta to products for the Store on .com experience.
+ * See also wc-calypso-bridge-jetpack-sync.php
+ *
+ * @since 0.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+function wc_calypso_bridge_products_get_context() {
+	if ( defined( 'DOING_AJAX' ) && DOING_AJAX ) {
+		$referrer = isset( $_SERVER['HTTP_REFERER'] ) ? $_SERVER['HTTP_REFERER'] : '';
+		$known_referrers = array(
```

Comment:
These equal signs should be aligned per the WP PHP coding standards.

Reply:
Aligned in https://github.com/Automattic/wc-calypso-bridge/pull/32/commits/e69a0aecbdc223db09d4f0e04add5162742dc5b4